mat-process-api.Tests/V1/Validators/GetProcessImageRequestValidatorTests.cs

```diff
@@ -20,6 +20,62 @@ public void SetUp()
             _getValidator = new GetProcessImageRequestValidator();
         }
 
+        #region Field is required
+
+        [TestCase(null)]
+        [TestCase("")]
+        [TestCase(" ")]
+        public void given_a_request_with_null_empty_or_whitespace_processRef_when_getProcessImageRequestValidator_is_called_then_it_returns_an_error(string processRef)
```

Comment:
As discussed earlier, null should not be tested as it will fire a 404 error. It's empty/whitespace possibility that we should validate against.

Reply:
It won't fire a 404 error because the .NET translates the %20 into null within a request object.
Here is the screenshot of debugging the request like that:

![image](https://user-images.githubusercontent.com/43747286/72914781-2155fb00-3d37-11ea-8cf9-d88b08439526.png)